tensorflow/compiler/mlir/hlo/tests/disc_ral_inject_execution_context.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: mlir-hlo-opt -disc-ral-inject-execution-context=entry-func-name=test \
+// RUN:  -canonicalize -lhlo-legalize-tensor-load-op %s -o - | FileCheck %s
```

Comment:
Do you need the `-canonicalize -lhlo-legalize-tensor-load-op` part here?

Reply:
removed `-lhlo-legalize-tensor-load-op`